lib/Helper/Configs/next.config.1-merged.js

```diff
@@ -0,0 +1,18 @@
+// This file sets a custom webpack configuration to use your Next.js app
```

Comment:
Can we rename the folder these test templates are located to something like `test-fixtures`?